### ISSUE TITLE
Have a problem using the np.array method

### ISSUE THREAD

**Issue by carl2099** [SciSharp/Keras.NET#245]
Hello 

I have a problem while using the np.array method

var inputArr = np.array(input);
var outputArr = np.array(output);

I am tring to transfer input and output into numpy array by using np.array method,in which input is double[365, 3] and output is double[365, 1].

However, outputArr is successfully transfered,but inputarr is null,I think it maybe means np.array method is not worked in double[365, 3] in some sort of reason.

I`d like to know how to solve it.

Thanks!

**Comment by deepakkumar1984** [comment]
Can you give me a sample code to test it out?

**Comment by carl2099** [comment]
sure,here is my code:

// Divide xinput and yinput
             int inputCols = 3;
             int outputCols = 1;
             int numRows = dt_select.Rows.Count;
             double[,] input = new double[numRows, inputCols];
             double[,] output = new double[numRows, outputCols];

             for (int i = 0; i < numRows; i++)
             {
                 for (int j = 0; j < inputCols; j++)
                 {
                     input[i, j] = Convert.ToDouble(dt_select.Rows[i][j]);
                 }
                 output[i, 0] = Convert.ToDouble(dt_select.Rows[i][dt_select.Columns.Count - 1]);
             }

             // Convert input and output arrays to numpy arrays
             NDarray inputArr = np.array(input);
             NDarray outputArr = np.array(output);

dt_select is a datatable with four columns. I use the first three columns as input and the fourth column as output, but can I show my data here? You may not be able to run this code directly.

**Comment by carl2099** [comment]
I converted the three columns of input into np.array and then combined them using np.concatenate, which solved the problem I mentioned, although not directly. But I ran into another problem.

Here is my code：

//Convert input and output arrays to numpy arrays respectively
NDarray inputArr1 = np.array(input1);
NDarray inputArr2 = np.array(input2);
NDarray inputArr3 = np.array(input3);
NDarray outputArr = np.array(output);

// Merge the input arrays column by column into one array
NDarray inputArr = np. concatenate(new NDarray[] { inputArr1, inputArr2, inputArr3 }, axis: 1);

// Reshape the input array to match the expected input shape of the keras model
NDarray inputArrReshape = inputArr.reshape(numRows, inputCols, 1) as NDarray<double>;

The return value of Numpy.NDarray.reshape is the reshaped array, but inputArrReshape is null. I don't understand why the value cannot be passed.
